### DISCUSSION TITLE
Had a bit of confusion getting setup

### DISCUSSION THREAD

**Discussion by cuppajoeman** [nvim-treesitter/nvim-treesitter#1421] [Ideas]
Hi as a new user to nvim-treesitter I just wanted to share some of my experience trying to install the python parser. 

When I saw this code block in the install guide`:TSInstall {language}` the highlighting lead me to believe I should write `:TSInstall {python}`, dumb on my end I know, but I had to come back 30min later to realize that.

It could be useful to make that a little more clear. 

Best,
C

**Comment by stsewd** [comment]
Thanks for the feedback, using `{foo}` is used in several programs to indicate a placeholder, we could add an example to make it more clear.
